driver/config/config_test.go

```diff
@@ -867,34 +888,38 @@ func TestLoadingTLSConfig(t *testing.T) {
 		p := config.MustNew(t, logger, os.Stderr, configx.SkipValidation())
 		p.MustSet(ctx, config.ViperKeyAdminTLSKeyPath, keyPath)
 		p.MustSet(ctx, config.ViperKeyAdminTLSCertPath, certPath)
-		assert.NotNil(t, p.GetTSLCertificatesForAdmin(ctx))
-		assert.Equal(t, "Setting up HTTPS for admin", hook.LastEntry().Message)
+		assert.NotNil(t, p.GetTLSCertificatesForAdmin(ctx))
+		assert.Equal(t, "Setting up HTTPS for admin (automatic certificate reloading active)", hook.LastEntry().Message)
```

Comment:
You could also use
```suggestion
		assert.Contains(t, hook.LastEntry().Message, "Setting up HTTPS for admin", hook.LastEntry().Message)
```
just FYI